plugins/chat/app/queries/chat/thread_unreads_query.rb

```diff
@@ -59,7 +59,21 @@ def self.call(
           AND user_chat_channel_memberships.muted = false
           AND user_chat_channel_memberships.user_id = :user_id
         ) AS unread_count,
-        0 as mention_count,
```

Comment:
Adding a mention count here so we can show the urgent badge the channel threads list and my threads list.